next.go

```diff
@@ -116,20 +173,46 @@ func bump(ref time.Time, pos int, reverse bool) time.Time {
 	if reverse {
 		factor = -1
 	}
+	loc := ref.Location()
 
 	switch pos {
 	case 0:
 		ref = ref.Add(time.Duration(factor) * time.Second)
 	case 1:
-		ref = ref.Add(time.Duration(factor) * time.Minute)
+		minTime := ref.Add(time.Duration(factor) * time.Minute)
```

Comment:
since now reverse bump is quite different, i think it is neat to use different func for bump() and bumpReverse()